### DISCUSSION TITLE
Recommended Server Spec

### DISCUSSION THREAD

**Discussion by yongra** [ucfcdl/UDOIT#864] [Q&A]
Hi everyone,

My school wants to use UDOIT.

I would like to get some server specs such as CPU, RAM, OS, and storage size. Does anyone have any recommendations?

Thank you,

Ra

**Comment by bagofarms** [comment]
We don't have recommended specs for CPU or RAM, but we recommend using Linux for the OS, such as Red Hat or Amazon Linux 2. At UCF, we're using a t2.micro instance on AWS, which only has 1 GiB of RAM and 1 virtual CPU. Hopefully that gives you a good idea.

**Comment by yongra** [comment]
Thank you for the info. One more question, how much disk space do you recommend, then?


Ra

**Reply by bagofarms** [reply]
Ours has 8 GiB of storage for the web server, but the database is handled through Amazon RDS, so we don't have a set size for that.
